ci/requirements_files.py

```diff
@@ -0,0 +1,13 @@
+"""Determine the Python requirements files."""
+
+from pathlib import Path
+
+def requirements_files() -> list[str]:
+    """Return the Python requirements files in the requirements directory."""
+    # We never return the internal requirements file, because it does not need to be checked
```

Comment:
This sounds like it's not in accordance with the docstring. We're only interested in requirements which are checked by ci?
Perhaps separate the logic to two methods, for installing from requirements and checking them?

Reply:
Hmm, yeah. I'll have to think about this one.

Reply:
It's not needed. The internal requirements are all off the form `-e ../shared_code` so they never need to be compiled nor checked. I've added a comment.